app/views/api/v2/authorizable.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @resource
+
+if @object.try(:class).try(:ancestors).to_a.include?(Authorizable)
+    node(:canEdit) { |resource| resource.authorized?(:"#{:edit}_#{resource.class.name.pluralize.downcase}") } 
+    node(:canDelete) {|resource| resource.authorized?(:"#{:delete}_#{resource.class.name.pluralize.downcase}") } 
+    node(:canCreate) { |resource| resource.authorized?(:"#{:create}_#{resource.class.name.pluralize.downcase}") } 
```

Comment:
`can_create` might not make a lot of sense here. if the index action returns zero results you have no way to tell that the user is authorized to create more. :-)

Reply:
agreed, it should probably passed as a prop once we have a full page react component (so you know if to add the create link or not)